packages/openapi-ts/src/types/config.d.ts

```diff
@@ -198,6 +199,17 @@ interface Input {
       include?: ReadonlyArray<string>;
     };
   };
+  fix?: {
+    schema?: Record<
+      string,
+      (
+        schema:
+          | OpenApiSchemaObject.V2_0_X
+          | OpenApiSchemaObject.V4_0_X
```

Comment:
How is this not throwing anywhere 😳

```suggestion
          | OpenApiSchemaObject.V3_0_X
```

Reply:
Yeah, weird that it's not throwing! Fixed it anyway.